packages/lesswrong/lib/routes.ts

```diff
@@ -47,6 +47,63 @@ async function getPostPingbackBySlug(parsedUrl: RouterLocation, slug: string) {
 
 const postBackground = "white"
 
+const lw18ReviewPosts = [
```

Comment:
Style note: handling things with position in arrays like this, as opposed to objects with named fields, is usually bad (it's too easy to accidentally wind up with values misaligned and going into the wrong fields).

Reply:
Yeah, makes sense. I'm not a huge array fan, but Oli suggested arrays for this.